examples/services.ml

```diff
@@ -1,7 +1,7 @@
 open Lwt
-open V1_LWT
+open Mirage_types_lwt
```

Comment:
Not sure about that one. Should we try to use `Mirage_console_lwt.S` and `Mirage_stack.V4` there? I am not sure to remember what these examples are supposed to show anyway, so please do what you think is best :-)